jetbrains-core/tst/software/aws/toolkits/jetbrains/ui/ResourceSelectorTest.kt

```diff
@@ -204,4 +222,15 @@ class ResourceSelectorTest {
             }
         }
     }
+
+    companion object {
+        fun mockCred(id: String) = object : ToolkitCredentialsProvider() {
+            override val id = id
+            override val displayName = id
+
+            override fun resolveCredentials(): AwsCredentials {
+                TODO("not implemented")
```

Comment:
Remove the TODO and just do `throw NotImplementedError()` so we dont mess with our TODOs